draft-ietf-avtcore-rtp-sframe.md

```diff
@@ -207,10 +213,61 @@ a=rtpmap:100 H264/90000
 a=rtpmap:101 VP8/90000
 ~~~
 
+# SFrame SDP RTP Key Management
+
+When SFrame is used with SDP and specifies RTP as the media transport, an additional key derivation step MUST be applied to produce a unique key per SDP m= line using the SSRC.
```

Comment:
Key management is really an application specific decision, I am not sure we should use MUST here.
I would tend to use similar wording as https://www.rfc-editor.org/rfc/rfc9605.html#name-key-management to express what is expected to be done.